.github/workflows/dependency-review.yml

```diff
@@ -22,4 +22,3 @@ jobs:
           fail-on-severity: critical
           # compatible/incompatible licenses addressed here: https://www.apache.org/legal/resolved.html
           # find SPDX identifiers here: https://spdx.org/licenses/
-          deny-licenses: MS-LPL, BUSL-1.1, QPL-1.0, Sleepycat, SSPL-1.0, CPOL-1.02, AGPL-3.0, GPL-1.0+, BSD-4-Clause-UC, NPL-1.0, NPL-1.1, JSON
```

Comment:
did you mean to delete this?

Reply:
Yes (mentioned in the PR description), I was hoping to revisit in a follow-up PR. I'll throw some more attempts at it to see if I can fix it here.